images/gh-gl-sync/SpackCIBridge.py

```diff
@@ -601,13 +558,6 @@ def sync(self):
             # Get tags on GitHub.
             tags = self.list_github_tags()
 
-            # Retrieve PRs that have already been synced to GitLab.
-            synced_prs = self.get_synced_prs()
-
-            # Find closed PRs that are currently synced.
-            # These will be deleted from GitLab.
-            closed_refspecs = self.get_prs_to_delete(all_open_prs["pr_strings"], synced_prs)
```

Comment:
@kwryankrattiger can you show me the new spackbot code that deletes branches from GitLab when a PR is merged on GitHub?

Reply:
Here is the line the operation is handled:

https://github.com/spack/spackbot/pull/68/files#diff-b8354323f0494dadd3ec1c8a317ce0f3f685d3ca78be54403f325d1ad0101580R61

And here is where it is called.

https://github.com/spack/spackbot/pull/68/files#diff-9afe3332bf5162c85164c3a680f549ee6c26428ffded12cf84de73e93ccce62eR142